shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-postgresql/src/main/antlr4/imports/postgresql/DDLStatement.g4

```diff
@@ -561,6 +562,12 @@ alterTableCmd
     | alterGenericOptions
     ;
 
+columnCompression
+    : COMPRESSION colId
+    | COMPRESSION DEFAULT
+    ;
+    
```

Comment:
@tuichenchuxin Please remove useless blank line.

Reply:
@strongduanmu Thanks for reminding, I have remove it.